disablestrongnamevalidation.ps1

```diff
@@ -0,0 +1,5 @@
+Set-Location HKLM:
+$registryPath = "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\.NETFramework"
+$Name = "AllowStrongNameBypass"
```

Comment:
do we need it for a specific assembly or entire machine? I just afraid it may create a security vulnerability on the machine.